Calculate/Calculate/Calculator.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Calculate
+{
+    public class Calculator
+    {
+
+        public Calculator() 
+        {
+            MathematicalOperations = new Dictionary<char, Func<float, float, float>>();
+            ((Dictionary<char, Func<float, float, float>>)MathematicalOperations).Add('+', Add);
+            ((Dictionary<char, Func<float, float, float>>)MathematicalOperations).Add('-', Subtract);
+            ((Dictionary<char, Func<float, float, float>>)MathematicalOperations).Add('*', Multiply);
+            ((Dictionary<char, Func<float, float, float>>)MathematicalOperations).Add('/', Divide);
+        }
+        public IReadOnlyDictionary<char, Func<float, float, float>>? MathematicalOperations;
+        public static float Add(float leftOperand, float rightOperand) 
+        {
+            return leftOperand + rightOperand;
+        }
```

Comment:
```suggestion
        public static float Add(float leftOperand, float rightOperand) => leftOperand + rightOperand;
```
You can shorten thses!